src/main/routes/userprofiles.ts

```diff
@@ -1,23 +1,23 @@
-import * as express from "express";
+
 import { fetchUsers } from "../service/user.profiles.service";
 import { Jurisdiction } from "../domain/jurisdiction";
 import { Validator } from "../validators/validate";
 
-const router = express.Router();
+const router = require("../routes/home");
 
 // Validate
-router.use((req, res, next) => {
+function validate(req, res, next) {
```

Comment:
Prefer to use the double-arrow (`=>`) notation to declare a function. This might fail the linting checks in the build pipeline.